Sources/ModernSlider/ModernSlider.swift

```diff
@@ -152,15 +157,24 @@ private struct SliderView: View {
 
     private var dragGesture: some Gesture {
         DragGesture(minimumDistance: 0)
-            .onChanged { value in
-                isDragging = true
-                updateOffset(at: value.location.x)
-                onChange()
-            }
-            .onEnded { _ in
+            .updating($dragOffsetY) { (value, offsetY, transaction) in
+                offsetY = value.location.x
+                updateOffset(at: offsetY)
+                isTapped = true
                 isDragging = false
```

Comment:
I think this should be set to `true` here instead since it's now in the `updating` modifier